packages/fuels-test-helpers/src/wallets_config.rs

```diff
@@ -0,0 +1,34 @@
+pub const DEFAULT_COIN_AMOUNT: u64 = 1_000_000_000;
```

Comment:
duplication with the one from `constants.rs` that was moved into `contract.rs`

Reply:
IMO these two should be kept separated. This one is the default parameter for the wallet setup config while the other one is   for querying UTXOs.

Reply:
This should be documented.